src/store/actionsV2/__tests__/getDiseasesForTypeAheadAction.test.js

```diff
@@ -0,0 +1,29 @@
+import { ACTIVE_TRIAL_STATUSES } from '../../../constants';
+import { getDiseasesForTypeAheadAction } from '../getDiseasesForTypeAheadAction';
+
+describe('getDiseasesForTypeAheadAction', () => {
+	it('should match returned object', () => {
+		const searchText = 'Breast Cancer';
+
+		const requestQuery = {
+			name: searchText,
+			current_trial_status: ACTIVE_TRIAL_STATUSES,
+			size: 10,
+			type: ['maintype', 'subtype', 'stage', 'grade'],
```

Comment:
one more thing - is 'grade' even a type? Old action only had ['maintype', 'subtype', 'stage']

Reply:
I was going by what was in the PP document for v2 request which had `grade` as a type.

Reply:
@kate-mashkina, @sarinapadilla confirmed with @bryanpizzillo Type `grade` shouldn't be used for CTS as its intersects with type `stage` and causes confusion.

Removed type `grade` from `v2` request for PR.